spec/factories/access_requests.rb

```diff
@@ -0,0 +1,29 @@
+# == Schema Information
+#
+# Table name: access_request
+#
+#  id               :integer          not null, primary key
+#  email_address    :text
+#  first_name       :text
+#  last_name        :text
+#  organisation     :text
+#  reason           :text
+#  request_date_utc :datetime         not null
+#  requester_id     :integer
+#  status           :integer          not null
+#  requester_email  :text
+#
+
+FactoryBot.define do
+  factory :access_request do
+    requester { build(:user, :opted_in) }
+    status { :requested }
+    request_date_utc { Time.now.utc }
+    email_address { Faker::Internet.email }
+    first_name { requester.first_name }
```

Comment:
```suggestion
    first_name { Faker::Name.first_name }
```

I think its should be the new user not the person requesting